manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -139,3 +166,10 @@ spec:
         hostPath:
           path: /etc/kubernetes
           type: Directory
+      - configMap:
+          name: kube-rbac-proxy
+        name: auth-proxy-config
+      - name: cloud-controller-manager-operator-tls
+        secret:
+          secretName: cloud-controller-manager-operator-tls
+          optional: true
```

Comment:
Nit, should have new line character at the end

Reply:
Done!